bin/cli.js

```diff
@@ -183,30 +183,67 @@ const handleDeprecatedEnvVariables = options => {
   ) {
     uiDeprecatedTag(
       i18n(`${i18nKey}.handleDeprecatedEnvVariables.portalEnvVarDeprecated`, {
-        configPath: getConfigPath(),
+        configPath: getConfigFilePath(),
       })
     );
     process.env.HUBSPOT_ACCOUNT_ID = process.env.HUBSPOT_PORTAL_ID;
   }
 };
 
+function addConfigEnvironmentVariablesMiddleware(options) {
```

Comment:
Not a blocker, but we might have enough middleware at this point to warrant moving it all into a dedicated middleware util file

Reply:
Agreed. Might wait until this is all merged though because that'll be one more thing that creates conflicts